pkg/database/dgraph.go

```diff
@@ -327,6 +327,26 @@ func dbInsert(c *client.Dgraph, data interface{}) (string, error) {
 	return uid, nil
 }
 
+// DBDelete deletes nodes from the db.
+// The data should be JSON marshalable
+func DBDelete(db *DataBase, nodeUID string) (string, error) {
```

Comment:
Delete should be enough DBDelete is tautological as this is in the database package